recipes/cyrsoxs/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "cyrsoxs" %}
+{% set version = "1.1.5.0" %}
+{% set sha = "7ce6e64d5e794751476e6e1b6dbb1cd940b2dd26796b8d416fd82b6c20302ff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/cyrsoxs/archive/refs/tags/V{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  skip: True  # [py<37]
+  skip: True  # [not linux64]
+  skip: True  # [cuda_compiler_version == "None"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make
+    - cudatoolkit
```

Comment:
```suggestion
```
I don't think you need to specify this. it will find the right one.

Instead we often specifcy other libraries
https://github.com/conda-forge/tensorflow-feedstock/blob/main/recipe/meta.yaml#L56

```
  host:
    # GPU requirements for CUDA
    - cudnn      # [cuda_compiler_version != "None"]
    - nccl       # [cuda_compiler_version != "None"]
```

Reply:
Removed, build still works.  The CUDA dependencies for this are entirely within the toolkit.